html_css/project_positioning.md

```diff
@@ -27,6 +27,7 @@ Send us your solution so we can show others! Submit a link to the Github [repo](
   <summary> Show Student Solutions </summary>
 
 * Add your solution below this line!
+* [kaliberpoziomka's solution](https://github.com/kaliberpoziomka/nytimesArticle) - [View in Browser](https://kaliberpoziomka.github.io/nytimesArticle/)
```

Comment:
[Simonoob's solution](https://github.com/Simonoob/rock-paper-scissors-game) -[View in Browser](https://codepen.io/SimoneBu/full/ExVeVWO)